src/main/java/frc/utils/applicationsutils/OnDeploy.java

```diff
@@ -0,0 +1,21 @@
+package frc.utils.applicationsutils;
+
+import javax.swing.*;
+
+public class OnDeploy {
+
+
+    public static void main(String[] args) {
+        startupLogic();
+    }
+
+    private static void startupLogic() {
+        //to be continued...
```

Comment:
finish before merging

Reply:
this is for future branches. there is nothing for me to add in this branch.